smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/integration/httpResponse/HTTPResponseBindingErrorGenerator.kt

```diff
@@ -83,6 +85,7 @@ class HTTPResponseBindingErrorGenerator(
                         .map { ctx.model.expectShape(it) as StructureShape }
                         .toSet()
                         .sorted()
+                    writer.addImport(SwiftSymbol.make("ClientRuntime", null, SwiftDependency.CLIENT_RUNTIME, "SmithyReadWrite"))
```

Comment:
An import of all of `ClientRuntime` with the `SmithyReadWrite` SPI annotation is required to access the `HTTPResponse.data()` instance method.